### PR TITLE
[FLINK-12894][doc][Chinese-translation]add Chinese documentation of how to configure and use catalogs in SQL CLI

### PR DESCRIPTION
**Pull request by yiduwangkai**
## What is the purpose of the change

*Support Chinese version of sqlClient.md*

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluser with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/dev/table/catalog.md [hunk 1]
```diff
@@ -190,9 +190,9 @@ Note that we only cover most commonly used data types for now.
 
 The following limitations in Hive's data types impact the mapping between Flink and Hive:
 
-\* maximum length is 255
+\* minimum length is 255
```
Comment (bowenli86) [inline docs/dev/table/catalog.md]: nice catch!

File: docs/dev/table/catalog.zh.md [hunk 2]
```diff
@@ -1,363 +0,0 @@
----
```
Comment (bowenli86) [inline docs/dev/table/catalog.zh.md]: revert changes to this file

File: docs/dev/table/sqlClient.zh.md [hunk 3]
```diff
@@ -24,71 +24,73 @@ under the License.
...
+Flink的Table & SQL API可以处理SQL语言编写的查询，但这些查询需要嵌入用Java或Scala编写的程序中。 而且，这些程序需要在提交到集群之前使用构建工具打包。 这或多或少地限制了Flink只能为Java/Scala程序员而使用。
 
-The *SQL Client* aims to provide an easy way of writing, debugging, and submitting table programs to a Flink cluster without a single line of Java or Scala code. The *SQL Client CLI* allows for retrieving and visualizing real-time results from the running distributed application on the command line.
+*SQL Client*旨在提供一种简单的方法来编写，调试和提交表程序到Flink集群，而无需单独编写Java或Scala代码。 *SQL Client CLI*允许在命令行上提交分布式应用程序来获得可视化的实时结果。
 
-<a href="{{ site.baseurl }}/fig/sql_client_demo.gif"><img class="offset" src="{{ site.baseurl }}/fig/sql_client_demo.gif" alt="Animated demo of the Flink SQL Client CLI running table programs on a cluster" width="80%" /></a>
+<a href="{{ site.baseurl }}/fig/sql_client_demo.gif"><img class="offset" src="{{ site.baseurl }}/fig/sql_client_demo.gif" alt="在集群上运行表程序的Flink SQL Client CLI的动画演示" width="80%" /></a>
+
+<span class="label label-danger">注意</span> SQL客户端处于早期开发阶段。 即使应用程序还没有生产就绪，它也可以是一个非常有用的工具，用于原型设计和使用Flink SQL。 在未来，社区计划通过提供基于REST的功能来扩展其功能 [SQL Client Gateway](sqlClient.html#limitations--future).
```
Comment (bowenli86) [inline docs/dev/table/sqlClient.zh.md]:
```suggestion
<span class="label label-danger">注意</span> SQL客户端处于早期开发阶段。 尽管它还没有生产就绪，它也可以是一个非常有用的工具，用于开发原型和测试Flink SQL。 在未来，社区计划通过提供基于REST的功能来扩展其功能 [SQL Client Gateway](sqlClient.html#limitations--future).
```

File: docs/dev/table/sqlClient.zh.md [hunk 4]
```diff
@@ -24,71 +24,73 @@ under the License.
...
 
 {% highlight bash %}
 ./bin/start-cluster.sh
 {% endhighlight %}
 
-### Starting the SQL Client CLI
+### 启动SQL客户端CLI
 
-The SQL Client scripts are also located in the binary directory of Flink. [In the future](sqlClient.html#limitations--future), a user will have two possibilities of starting the SQL Client CLI either by starting an embedded standalone process or by connecting to a remote SQL Client Gateway. At the moment only the `embedded` mode is supported. You can start the CLI by calling:
+SQL客户端脚本也位于Flink的二进制目录中。 [将来](sqlClient.html#limitations--future)，用户可以通过启动嵌入式独立进程或连接到远程SQL客户端网关来启动SQL Client CLI。 目前只支持`embedded`模式。 您可以通过调用以下命令启动CLI：
```
Comment (bowenli86) [inline docs/dev/table/sqlClient.zh.md]: probably should be:

```suggestion
SQL客户端脚本也位于Flink的二进制目录中。 [将来](sqlClient_zh.html#局限与未来)，用户可以通过启动嵌入式独立进程或连接到远程SQL客户端网关来启动SQL Client CLI。 目前只支持`embedded`模式。 您可以通过调用以下命令启动CLI：
```

**Comment by yiduwangkai** [comment]
i have updated @bowenli86

**Comment by wuchong** [comment]
@yiduwangkai , please rebase your commit on the latest master and do not use "merge".

**Comment by yiduwangkai** [comment]
yes i used rebase at first, But when I execute this command, when I push,
git asks for pull, maybe it is improper operation before, causing the
situation to be merged when the pull occurs, but please be assured that no
other files have been changed. thx

Jark Wu <notifications@github.com> 于2019年7月21日周日 下午10:19写道：

**Comment by yiduwangkai** [comment]
> @yiduwangkai , please rebase your commit on the latest master and do not use "merge".

yes i used rebase at first, But when I execute this command, when I push, git asks for pull, maybe it is improper operation before, causing the situation to be merged when the pull occurs, but please be assured that no other files have been changed. thx

**Comment by yiduwangkai** [comment]
Why is there a new merge commit extra information, because I use the pull
command to pull the latest code of the upstream branch, this record will be
generated if the local branch lags behind the upstream branch

kai wang <yiduwangkai@gmail.com> 于2019年7月21日周日 下午11:28写道：

**Comment by bowenli86** [comment]
Hi @yiduwangkai , can you rebase this PR?

**Comment by yiduwangkai** [comment]
> Hi @yiduwangkai , can you rebase this PR?

Ok

**Comment by yiduwangkai** [comment]
i want to close this pr, merge [apache/flink] [FLINK-13086]add Chinese
documentation for catalogs (#9163)
and [apache/flink] [FLINK-12894]add Chinese documentation of how to
configure and use catalogs in SQL CLI (#9164) together and submit a new
one, do you agree

Bowen L <notifications@github.com> 于2019年8月13日周二 上午2:36写道：

**Pull request closed without merging**